src/app.ts

```diff
@@ -0,0 +1,61 @@
+import { MongoDriverFactory } from "./MongoConnectorFactory";
+import { MongoDriver } from "./MongoConnector";
+import * as body from "body-parser";
+import express from "express";
+import { Console } from "console";
+
+const PORT = 3000;
+const app = express();
+
+app.use(body.json());
+
+let taskDataStore;
+MongoDriverFactory.build()
+  .then(async (datastore) => {
+    taskDataStore = datastore;
+  })
+  .catch(e => {
+    throw e;
+  });
+
+  //Gets All Tasks
+app.get("/", (req,res)=> {
```

Comment:
So for this route I would suggest /tasks so that if for example we wanted to add a new resource like categories we can have a /categories route which would return the categories.